coderd/database/queries/provisionerjobs.sql

```diff
@@ -144,3 +144,15 @@ WHERE
 	updated_at < $1
 	AND started_at IS NOT NULL
 	AND completed_at IS NULL;
+
+-- name: InsertProvisionerJobTimings :many
+INSERT INTO provisioner_job_timings (job_id, started_at, ended_at, stage, source, action, resource)
```

Comment:
how many rows do we expect to have, on average, per build?

Reply:
How long is a piece of string? 😛 
Depends on the number of resources. I'll pull some numbers from the existing logs next week to be sure, but I'd imagine it'll be in the 10-100 range.

Reply:
Gotcha. We may need to think about pruning old workspace builds at some point; we currently have 32,631 rows in our dev db starting from 2022-08-10.